src/methods/convex_hull.jl

```diff
@@ -0,0 +1,92 @@
+#=
+# Convex hull
+
+The _convex hull_ of a set of points is the smallest **convex** polygon that contains all the points.
+
+GeometryOps.jl provides a number of methods for computing the convex hull of a set of points, usually
+linked to other Julia packages.  
+
+## Example
+
+### Simple hull
+```@example simple
+import GeometryOps as GO, GeoInterface as GI
+using CairoMakie # to plot
+
+points = randn(GO.Point2f, 100)
+f, a, p = plot(points; label = "Points")
+hull_poly = GO.convex_hull(points)
+lines!(a, hull_poly; label = "Convex hull", color = Makie.wong_colors()[2])
+axislegend(a)
+f
+```
+
+## Convex hull of the USA
+```@example usa
+import GeometryOps as GO, GeoInterface as GI
+using CairoMakie # to plot
+using NaturalEarth # for data
+
+all_adm0 = naturalearth("admin_0_countries", 110)
+usa = all_adm0.geometry[findfirst(==("USA"), all_adm0.ADM0_A3)]
+f, a, p = lines(usa)
+lines!(a, GO.convex_hull(usa); color = Makie.wong_colors()[2])
+f
+```
+
+=#
+
+"""
+    convex_hull([method], geometries)
+
+Compute the convex hull of the points in `geometries`.  
+Returns a `GI.Polygon` representing the convex hull.
+
+Note that all 
+
+!!! warning
+    This interface only computes the 2-dimensional convex hull!
+
+    For higher dimensional hulls, use the relevant package (Qhull.jl or similar).
+"""
+function convex_hull end
+
+"""
+    DelaunayTriangulationMethod()
+
+This is an algorithm for the [`convex_hull`](@ref) function.
+
+Uses [`DelaunayTriangulation.jl`](https://github.com/JuliaGeometry/DelaunayTriangulation.jl) to compute the convex hull.
+This is a pure Julia algorithm which provides an optimal Delaunay triangulation.
+
+See also [`convex_hull`](@ref)
+"""
+struct DelaunayTriangulationMethod end
```

Comment:
```suggestion
struct MonotoneChainMethod end
```

`DelaunayTriangulationMethod` sounds like you're doing `get_convex_hull(triangulate(points))` (i.e. getting the boundary of the unconstrained Delaunay triangulation) when really you're just doing the monotone chain algorithm which happens to be in DelaunayTriangulation.jl